src/components/Navbar.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Please implement the Navbar component having Links to display the Home and About Page